situp-plugins/apmtracesource/build.gradle

```diff
@@ -7,6 +7,20 @@ dependencies {
     compile project(':situp-api')
     compile project(':situp-plugins:common')
     implementation 'io.netty:netty-all:4.1.51.Final'
+    implementation 'io.opentelemetry:opentelemetry-proto:0.6.0'
+    implementation 'com.google.protobuf:protobuf-java-util:3.11.0'
     implementation "com.fasterxml.jackson.core:jackson-databind:${versionMap.jackson_databind}"
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${versionMap.jackson_dataformat_yaml}"
+    testImplementation 'org.assertj:assertj-core:3.6.2'
+    testImplementation(platform('org.junit:junit-bom:5.7.0'))
+    testImplementation('org.junit.jupiter:junit-jupiter')
+
+//    testImplementation "junit:junit:${versionMap.junit}"
+//    testImplementation "org.junit.jupiter:junit-jupiter-api:5.5.2"
+//    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.5.2"
+//    testRuntime "org.junit.platform:junit-platform-runner:1.5.2"
```

Comment:
Do we need this now?

Reply:
Not needed my bad, deleted the commented out bit now.